storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/operations/delete/PreparedDeleteByQuery.java

```diff
@@ -102,16 +102,16 @@ public DeleteResult executeAsBlocking() {
     public Observable<DeleteResult> asRxObservable() {
         throwExceptionIfRxJavaIsNotAvailable("asRxObservable()");
 
-        return Observable
-                .create(OnSubscribeExecuteAsBlocking.newInstance(this))
-                .subscribeOn(Schedulers.io());
+        final Observable<DeleteResult> observable =
+                Observable.create(OnSubscribeExecuteAsBlocking.newInstance(this));
+        return RxJavaUtils.subscribeOn(observable, storIOContentResolver);
```

Comment:
Can't you create a function that will create `Observable` from current operation and apply scheduler? Will save 2 lines in each operation.

Same is applicable for `Completable` and `Single`, I believe :)